README.md

```diff
@@ -4,6 +4,43 @@ react-native-fabric-crashlytics
 
 Reports javascript exceptions in React Native to the Crashlytics server, using the react-native-fabric library.
 
+Dependencies
+-----
+
+You need to install react-native-fs and link it to your main project to run this:
+
+```
+npm install -S react-native-fs
```

Comment:
Why aren't you listing this as a dependency of this project itself, versus requiring users to install it directly?

Similarly, you should be able to setup this project's rnpm-link to call that project's rnpm-link, right? (So that the instructions would just say to `react-native link react-native-fabric-crashlytics`?)

Reply:
Well, for that I would need to add the iOS and Android folders with the linking parts of react-native-fs itself on the project and I never did that before. Also, I was out of scope already for the project that needed this. Anyway, if someone has experience on that part, it would be better indeed.

Reply:
Fair :)